lib/calendar.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+class CalendarEvent {
+  static toICS(json) {
+    const data = JSON.parse(json);
+    const handleBlock = (block, result = []) => {
```

Comment:
This method should be extracted from class because there are no reasons to hold it in static method closure (no this or other class-related logic)